src/ralph/data_center/admin.py

```diff
@@ -595,6 +598,36 @@ def show_location(self, obj):
     show_location.short_description = _('Location')
     show_location.allow_tags = True
 
+    def scan_status(self, obj):
```

Comment:
* extract this to mixin
* mix it in:
  * DataCenterAsset
  * CloudHost
  * VirtualServer
  * DCHost
* this way will make extra query (at least one) for each row in the list - we have to make it better (with constant number of queries :))